app.rb

```diff
@@ -22,6 +22,13 @@ class CondaAPI < Sinatra::Base
     Conda.instance.find_package(params["name"]).to_json
   end
 
+  get "/package/:name/:version" do
+    content_type :json
+    package = Conda.instance.find_package(params["name"])
+    package[:versions] = package[:versions]&.select { |version| version[:number] == params["version"] }
+    package.to_json
```

Comment:
Is it ok to send a null back if we don't find it?

Reply:
Hmm actually that'll explode.

Reply:
send null back? It will 404 if we don't find the package

Reply:
Ah right! Cool.